Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs

```diff
@@ -325,7 +325,8 @@ eval (MarkerLabelOnClick markerName) state =
     let isSource = Just (markerName == "ny_ic_src_marker")
     let updatedState = state{props{isSource = isSource, hasEstimateBackpoint = true, currentStage = SearchLocationModel}}
     continue updatedState
-  else if (any (_ == state.props.currentStage) [ RideAccepted, ChatWithDriver]) then continueWithCmd state [pure $ EditLocation ""]
+  else if (any (_ == state.props.currentStage) [ RideAccepted, ChatWithDriver]) && state.data.config.feature.enableEditPickupLocation then continueWithCmd state [pure $ EditLocation ""]
+  else if state.props.currentStage == RideStarted && state.data.config.feature.enableEditDestination then continueWithCmd state [pure $ EditingDestination]
```

Comment:
RideAccepted Stage also

Reply:
At RideAccepted stage we dont show source to destination, we show route of driver to pickup

Reply:
Still, in this action you can't decide just on behalf of currentStage.